lib/withTypescript.js

```diff
@@ -9,7 +9,20 @@ module.exports = (base) => ({
 
     '@typescript-eslint/array-type': ['error', { default: 'generic' }],
     '@typescript-eslint/prefer-optional-chain': ['error'],
-
+    '@typescript-eslint/consistent-type-definitions': ['error', 'interface'],
+    '@typescript-eslint/consistent-type-imports': 'error',
+    '@typescript-eslint/no-floating-promises': 'error',
+    '@typescript-eslint/no-misused-promises': [
+      'error',
+      {
+        'checksVoidReturn': false,
+      },
+    ],
+    '@typescript-eslint/switch-exhaustiveness-check': 'warn',
+    '@typescript-eslint/no-empty-interface': 'error',
```

Comment:
Do you think it’s a problem? I find empty interfaces useful sometimes with extending — where something have same structure, but is used for different reasons. Looking at https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-empty-interface.md this would also be marked as an error?

Reply:
That's very true actually, now I think about it, I think we do this in Scrlly? I'll remove

Reply:
Not in scrolly, no.